conf/modules.config

```diff
@@ -928,25 +928,27 @@ process{
 // PAIR_VARIANT_CALLING
 
     //ASCAT
-    if (params.tools && params.tools.contains('ascat')) {
-        withName: 'ASCAT' {
-            ext.args = {
-                [
-                "gender": meta.sex,
-                //"genomeVersion": "hg19"
-                //"purity": (optional),
-                //"ploidy": params.ploidy,
-                //"gc_files": (optional),
-                //"minCounts": (optional),
-                //"chrom_names": (optional),
-                //"min_base_qual": (optional),
-                //"min_map_qual": (optional),
-                //"ref_fasta": (optional),
-                //"skip_allele_counting_tumour": (optional),
-                //"skip_allele_counting_normal": (optional)
-                ]
-            }
-        }
+    withName: 'ASCAT' {
+
+        ext.args = {[
+            "gender": meta.gender,
+            "genomeVersion": params.ascat_genome,
+            "purity": params.ascat_purity,
+            "ploidy": params.ploidy,
+            //"minCounts": (optional),
+            "chrom_names": 'c("21","22")' //,
+            //"min_base_qual": (optional),
```

Comment:
Seems like a good idea to add to the Paramus

Reply:
We could also just leave the default values for base and mapping quality and just add the chromosomes as a new parameter

Reply:
maybe lets leave the default for them all and also for chr. (worst case they can be overwritten with a custom config)